bundle/tests/environments_job_and_pipeline_test.go

```diff
@@ -15,7 +14,6 @@ func TestJobAndPipelineDevelopmentWithEnvironment(t *testing.T) {
 	assert.Len(t, b.Config.Resources.Pipelines, 1)
 
 	p := b.Config.Resources.Pipelines["nyc_taxi_pipeline"]
-	assert.Equal(t, "environments_job_and_pipeline/databricks.yml", filepath.ToSlash(p.ConfigFilePath))
```

Comment:
Can we still check that location of the resource correct but using dynamic value?